server/server.js

```diff
@@ -1,11 +1,40 @@
 const express = require('express');
-
 const app = express();
+
 const path = require('path');
+
+const cookies = require('cookie-session');
+const bodyParser = require('body-parser');
+
+//routers
+const login = require('./router/login');
+const signin = require('./router/signin');
+const aquarium = require('./router/aquarium');
+
 const port = 3000;
 
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded());
+
+
 app.use('/api', (req, res) => res.sendStatus(200));
 
+app.use('/login', login);
+
+app.use('/signin', signin);
+
+app.use('/aquarium', aquarium);
+
+if (process.env.NODE_ENV === 'production') {
+  app.use('/build', express.static(path.resolve(__dirname, '../build')));
+  app.get('/', (req, res) => {
+    res.sendFile(path.join(__dirname, '../client/index.html'));
+  });
+}
+
+
+
```

Comment:
Would be nice to add global error handling eventually and also send 404 error if route is not found